addons/mail/static/src/scss/res_config_settings_views.scss

```diff
@@ -0,0 +1,4 @@
+.o_settings_color_picker {
```

Comment:
Feel like this should not be done here (especially we avoi custom css in form views)